sos/report/plugins/python.py

```diff
@@ -30,7 +30,13 @@ def setup(self):
         self.add_cmd_output(
             self.python_version, suggest_filename="python-version"
         )
-        self.add_cmd_output("pip list")
+        pips = self.exec_cmd("whereis pip")
```

Comment:
This works fine on RHEL 7/8, however on Fedora 34 I get:

```
$ whereis pip
pip: /usr/bin/pip /usr/bin/pip3.9 /usr/share/man/man1/pip.1.gz
```

Using `whereis -b` though gives the expected output.

Reply:
Good point, PR updated.